creator-node/src/utils/validateAudiusUserMetadata.js

```diff
@@ -8,6 +8,7 @@ const validateAssociatedWallets = (metadataJSON) => {
   if ('user_id' in metadataJSON && 'associated_wallets' in metadataJSON) {
     const userId = metadataJSON['user_id']
     const walletMappings = metadataJSON['associated_wallets']
+    if (!walletMappings) return true
```

Comment:
I don't know if we need this if there's a check on line 8 for `'associated_wallets' in metadataJSON` 
I think I also updated in another pr to check for this value to be null.
I guess it can't hurt though. feel free to leave in.